angular-workspace/projects/ni/nimble-angular/label-providers/core/tests/nimble-label-provider-core-with-defaults.directive.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, ElementRef, ViewChild } from '@angular/core';
+import { TestBed } from '@angular/core/testing';
+import { CommonModule } from '@angular/common';
+import { computeMsgId } from '@angular/compiler';
+import { loadTranslations } from '@angular/localize';
+import type { LabelProviderCore } from '../nimble-label-provider-core.directive';
+import { NimbleLabelProviderCoreModule } from '../nimble-label-provider-core.module';
+
+describe('Nimble LabelProviderCore withDefaults directive', () => {
+    @Component({
+        template: `
+        <nimble-label-provider-core withDefaults #labelProvider>
+        </nimble-label-provider-core>
+         `
+    })
+    class TestHostComponent {
+        @ViewChild('labelProvider', { static: true }) public labelProvider: ElementRef<LabelProviderCore>;
+    }
+
+    let labelProvider: LabelProviderCore;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            declarations: [TestHostComponent],
+            imports: [NimbleLabelProviderCoreModule, CommonModule]
+        });
+    });
+
+    beforeEach(() => {
+        loadTranslations({
```

Comment:
Note: I got the general approach for this (using `loadTranslations` and `computeMsgId` to test `$localize`) from [some of Angular's autotests](https://github.com/angular/angular/blob/dd4361520267fb9fcd58e656f4637fdbc41a66f7/packages/core/test/acceptance/i18n_spec.ts)